api/src/db/models/transfer/transfer_topportunity_models.py

```diff
@@ -0,0 +1,33 @@
+from datetime import date
+
+from sqlalchemy import VARCHAR, Integer
+from sqlalchemy.orm import Mapped, mapped_column
+
+from src.db.models.base import Base, TimestampMixin
+
+
```

Comment:
Could we add a comment about why this table is a special case that uses `VARCHAR(length=...)` not `Text` (I'm assuming because it's a transfer table)?

Reply:
Yeah, I can add that, short version is this is trying to match the existing schema as closely as possible. If it turns out DMS has zero issue doing Oracle VARCHAR to Postgres TEXT, then I'll setup tables that way - but we need to verify that when testing first.